index.js

```diff
@@ -245,6 +250,17 @@ module.exports = class Hyperdrive extends ReadyResource {
     return await this.entry(name) !== null
   }
 
+  watch (folder) {
+    folder = normalizePath(folder || '/')
+
+    if (folder.endsWith('/')) folder = folder.slice(0, -1)
+
+    const range = this.sub.range({ gt: folder + '/', lt: folder + '0' })
+    const options = { map: this.checkout }
```

Comment:
needs a bind

Reply:
If currently it's not throwing it means it's fine?

Initially the code was:
https://github.com/holepunchto/hyperdrive-next/pull/34/commits/8d39ed3111c2588695b393b0ec88579ceeffa3d0#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R256

Later I changed it to current `{ map: this.checkout }` and it doesn't throw or anything, seems ok?